src/palm-react-native-ui-kit/components/ProfileHeader.tsx

```diff
@@ -5,85 +5,137 @@ import { ContractAddr, SupportedNetworkEnum } from 'palm-core/types'
 import { FormText, Row } from 'palm-react-native-ui-kit/components'
 import { useAppNavigation } from 'palm-react-native/app/useAppNavigation'
 import useProfile from 'palm-react/hooks/auth/useProfile'
-import React, { ReactElement } from 'react'
+import React, { ReactElement, useEffect } from 'react'
 import { useTranslation } from 'react-i18next'
 import { ImageBackground, Pressable, StyleSheet, View } from 'react-native'
-import Icon from 'react-native-vector-icons/Ionicons'
+import Ionicons from 'react-native-vector-icons/Ionicons'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
 
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
 import LensProfileHeaderSection from './LensProfileHeaderSection'
-import SupportedNetworkRow from './molecules/SupportedNetworkRow'
 import ProfileHeaderChatButton from './ProfileHeaderChatButton'
 import ProfileWalletAddress from './ProfileWalletAddress'
 import ProfileWalletBalances from './ProfileWalletBalances'
+import SupportedNetworkRow from './molecules/SupportedNetworkRow'
 import Avatar from './sendbird/Avatar'
 
+import { GroupChannel } from '@sendbird/chat/groupChannel'
+import { useSendbirdChat } from '@sendbird/uikit-react-native'
 export type ProfileHeaderProps = {
   userAddress?: ContractAddr
   userProfileId?: string
   isMyPage: boolean
+  channel?: GroupChannel
   selectedNetwork: SupportedNetworkEnum
   onNetworkSelected?: (selectedNetwork: SupportedNetworkEnum) => void
   onToggleShowUserTokensSheet?: () => void
+  onToggleChannelUserControl?: () => void
 }
 
 const ProfileHeader = React.memo(
   ({
     userAddress,
     userProfileId,
     isMyPage,
+    channel,
     selectedNetwork,
     onNetworkSelected,
     onToggleShowUserTokensSheet,
+    onToggleChannelUserControl,
   }: ProfileHeaderProps): ReactElement => {
     const { navigation } = useAppNavigation()
     const { t } = useTranslation()
 
+    const { sdk } = useSendbirdChat()
+
+    const [isOperator, setIsOperator] = React.useState<boolean>(false)
+    useEffect(() => {
+      const checkOperator = channel?.members.find(member => {
+        return (
+          member.role === 'operator' &&
+          member.userId === sdk?.currentUser?.userId
+        )
+      })
+      if (checkOperator) {
+        setIsOperator(true)
+      }
+    }, [])
```

Comment:
you don't need to use hook here. just use 

` channel.myRole === 'operator' `

Reply:
changed. thank you!